TSC-Charter.md

```diff
@@ -56,14 +57,29 @@ one-fourth of the TSC membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
 TSC members affiliated with the over-represented employer(s).
 
-The TSC members shall consist of Maintainers from Core Projects as
-defined in the project lifecycle document and Section 7.
+The TSC membership shall include (but is not limited to) Maintainers from Core
+Projects as defined in the project lifecycle document and Section 7. The TSC
+may, at its discretion, invite new members to join the TSC who are not
+considered to be Maintainers from Core Projects.
```

Comment:
This charter defines ["Collaborators" and "Contributors"](https://github.com/nodejs/TSC/issues/89#issuecomment-226394270) but not "Maintainers"... so let's avoid it. 

FYI: The By-Laws use "Maintainers" but doesn't define it either. [My suggested change to the By-Laws](https://github.com/nodejs/TSC/pull/142#issuecomment-248979400) would remove that term there too.

IMO, this paragraph doesn't bring any value. I'd just remove it and keep it simple since what it says above is adequate:
> The TSC may add additional members to the TSC by a standard TSC motion and vote.